ibm_mq/datadog_checks/ibm_mq/ibm_mq.py

```diff
@@ -25,6 +27,8 @@
 
 
 class IbmMqCheck(AgentCheck):
+    process_matcher = QueueManagerProcessMatcher()
```

Comment:
We are using limit=1 (default value). What about several checks running at the same time (with different check_id)? Is that case considered?

Reply:
Yes that's the purpose, we only want one running

Reply:
In that case the second check will be ignored?

Reply:
Yup!